cmake/ConfigOptions.cmake

```diff
@@ -164,6 +164,8 @@ option(USE_VERSION_FROM_GIT_TAG "Extract FreeRDP version from git tag." ON)
 option(WITH_CAIRO    "Use CAIRO image library for screen resizing" OFF)
 option(WITH_SWSCALE  "Use SWScale image library for screen resizing" OFF)
 
+option(WITH_UNICODE_BUILTIN  "Use built-in Unicode conversion (don't use system-provided libraries)" OFF)
```

Comment:
This option only affects `WinPR`, so it might be better to have that somewhere in `winpr` subdirectory instead of here.